util/orberrors/orberrors.go

```diff
@@ -0,0 +1,73 @@
+// Package orberrors provides an error that must be transported on client<->server operations.
+package orberrors
```

Comment:
Lets just call this errors

Reply:
"errors" is a go package, that's why not.